src/index.rst

```diff
@@ -92,17 +103,17 @@ Documentation
 Component Level Dependencies
 ============================
 
-Dependencies between components are as follows:
+Axom has the following inter-component dependencies:
 
-- Core has no dependencies, and the other components depend on Core
+- Core has no dependencies and the other components depend on Core
 - Slic optionally depends on Lumberjack
 - Slam, Spin, Primal, Mint, Quest, and Sidre depend on Slic
 - Mint optionally depends on Sidre
 - Quest depends on Slam, Spin, Primal, and Mint
-- Inlet depends on Sidre, and Slic
+- Inlet depends on Sidre, Slic, and Primal
 
-The figure below summarizes the dependencies between the components.  Solid links
-indicate hard dependencies; dashed links indicate optional dependencies.
+The figure below summarizes these dependencies. Solid links indicate hard 
+dependencies; dashed links indicate optional dependencies.
 
 .. graphviz:: docs/dependencies.dot
```

Comment:
Sorry can you change this line as well?  The image is now wrong as well...

https://github.com/LLNL/axom/blob/d8f5f41c830453a4ceba6711de90dc9f0f71adee/src/docs/dependencies.dot#L10

Reply:
Sure. Will do this tomorrow morning. We're arguing in the PCC meeting now....